extensions/libarchive/MergeContent.cpp

```diff
@@ -140,6 +146,10 @@ void MergeContent::onSchedule(core::ProcessContext *context, core::ProcessSessio
   if (context->getProperty(KeepPath.getName(), value) && !value.empty()) {
     org::apache::nifi::minifi::utils::StringUtils::StringToBool(value, keepPath_);
   }
+  value = "";
+  if (context->getProperty(AttributeStrategy.getName(), value) && !value.empty()) {
+      this->attributeStrategy_ = value;
```

Comment:
What is the point of having an explicit this?

Reply:
consistency, all others above this follow the same pattern, same for resetting the `value`, I'm not sure if removing the `this` improves the code or not

Reply:
We denote member variables with a trailing `_`. There is no added readibility in further specifying its explicitness.

Reply:
I think having the `this` keyword is just redundant here if we have the `_` suffix for member variables, and as resetting the `value` has no reason in this case I can just remove them in all occurrences and keep the consistency.